### ISSUE TITLE
Kraken2 custom database buildup failure?

### ISSUE THREAD

**Issue by jwsohn** [DerrickWood/kraken2#160]
Hello.

I was met with this error running `kraken2-build --download-taxonomy --db dbname` and it seems that the NCBI FTP server does not have the file in place. Could you have a look at this?

The error message was:

```
kraken2-build --download-taxonomy --db /usr/local/kraken2/viruses-db 
Downloading nucleotide est accession to taxon map...rsync: link_stat "/taxonomy/accession2taxid/nucl_est.accession2taxid.gz" (in pub) failed: No such file or directory (2)
```

When I FTP to the NCBI server (ftp.ncbi.nlm.nih.gov), I get the following directory structure:

```
$ ftp ftp.ncbi.nlm.nih.gov
Trying 130.14.250.11...
Connected to ftp.ncbi.nlm.nih.gov (130.14.250.11).
220 FTP Server ready.
Name (ftp.ncbi.nlm.nih.gov:jwsohn): ftp
331 Anonymous login ok, send your complete email address as your password
Password:
230 Anonymous access granted, restrictions apply
Remote system type is UNIX.
Using binary mode to transfer files.
ftp> cd pub/taxonomy/accession2taxid/
250 CWD command successful
ftp> dir
227 Entering Passive Mode (130,14,250,11,196,50).
150 Opening BINARY mode data connection for file list
-r--r--r--   1 ftp      anonymous     3095 Sep  9 15:20 README
-r--r--r--   1 ftp      anonymous 153702592 Sep  8 16:14 dead_nucl.accession2taxid.gz
-r--r--r--   1 ftp      anonymous       63 Sep  8 16:14 dead_nucl.accession2taxid.gz.md5
-r--r--r--   1 ftp      anonymous 625647620 Sep  8 16:14 dead_prot.accession2taxid.gz
-r--r--r--   1 ftp      anonymous       63 Sep  8 16:14 dead_prot.accession2taxid.gz.md5
-r--r--r--   1 ftp      anonymous 484997278 Sep  8 16:14 dead_wgs.accession2taxid.gz
-r--r--r--   1 ftp      anonymous       62 Sep  8 16:14 dead_wgs.accession2taxid.gz.md5
-r--r--r--   1 ftp      anonymous 1879108474 Sep  8 16:15 nucl_gb.accession2taxid.gz
-r--r--r--   1 ftp      anonymous       61 Sep  8 16:15 nucl_gb.accession2taxid.gz.md5
-r--r--r--   1 ftp      anonymous 3391558433 Sep  8 16:16 nucl_wgs.accession2taxid.gz
-r--r--r--   1 ftp      anonymous       62 Sep  8 16:16 nucl_wgs.accession2taxid.gz.md5
-r--r--r--   1 ftp      anonymous  3278110 Sep  8 16:16 pdb.accession2taxid.gz
-r--r--r--   1 ftp      anonymous       57 Sep  8 16:16 pdb.accession2taxid.gz.md5
-r--r--r--   1 ftp      anonymous 5501386302 Sep  8 16:18 prot.accession2taxid.gz
-r--r--r--   1 ftp      anonymous       58 Sep  8 16:18 prot.accession2taxid.gz.md5
226 Transfer complete
ftp>
```

Thanks,

Jung-woo

**Comment by tbigot** [comment]
Hi Jung-Woo,

It seems this issue was addressed with the commit 562c290. Switching to the git version of kraken2 worked for me.

**Comment by BenLangmead** [comment]
Thanks- closing for now
